src/ticket/ticket.entity.ts

```diff
@@ -0,0 +1,28 @@
+import {
+    Entity,
+    PrimaryGeneratedColumn,
+    Column,
+    ManyToOne,
+    JoinColumn,
+} from 'typeorm';
+
+import {User} from '../user/user.entity'
+
+@Entity('tickets') 
+export class Ticket {
```

Comment:
This is not in sync [the schema](https://github.com/Asociatia-AI3/difffusion-ticketing/blob/main/src/migrations/1746692591614-InitialMigration.ts#L11). We have Tickets and TicketUses, also see README; primary key should be [ulid](http://npmjs.com/package/ulid), not UUID - please adjust for users as well